apps/app-article-server/src/browser/components/mobile-article-list.js

```diff
@@ -64,14 +65,18 @@ const MobileList = (props) => {
                   {item.listImage === null ? (
                     <img className="listImage" src={hblDefaultImage} alt="" />
                   ) : (
-                    <img
-                      className="listImage"
+                    <>
+                    <LazyLoadImage
                       src={item.listImage.url.includes("imengine")
-                          ? item.listImage.url + "&function=hardcrop&width=360&height=200&q=65"
-                          : item.listImage.tinyThumb
-                          }
-                      alt=""
+                      ? item.listImage.url + "&function=hardcrop&width=360&height=200&q=65"
+                      : item.listImage.tinyThumb
+                      }
```

Comment:
I don't know what kind of JS style convention we're following, but my intuition tells to use indentation here.

Reply:
`npx prettier --write src/browser/components/mobile-article-list.js` formats stuff nicely.

Reply:
Good catch, I'll fix it.